components/FormStyles/TextArea.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components'
```

Comment:
the TextArea in this file is incredibly similar to the InputBox from components/FormStyles/InputBox.tsx -- consider having this inherit the styles from the InputBox instead of copying a bunch of the styling?

Reply:
highkey did not notice but they take in different props and i don't know how thrilled styled components would be abt that